cpp/arcticdb/column_store/column.hpp

```diff
@@ -936,17 +937,21 @@ class Column {
 
     std::optional<util::BitMagic> sparse_map_;
     util::MagicNum<'D', 'C', 'o', 'l'> magic_;
-}; //class Column
+};
 
-template <typename T>
-JiveTable create_jive_table(const Column& col) {
-    JiveTable output(col.row_count());
+template <typename TagType
+    >
+JiveTable create_jive_table(const Column& column) {
```

Comment:
Is there any performance benefit to having this as a separate implementation? Or could it just call the multiple-columns version with one element to the vector?

Reply:
I think it will be faster, in particular an unstable sort witll be faster than a stable one. Mostly I just expect them to diverge quite a bit more in the future - the multi-sort implementation is as discussed pretty basic and I imagine we will want to improve it without worrying about the single column imlementation